change/@fluentui-react-next-2020-05-27-17-09-55-spinbutton-reintroduce-wrong-behavior.json

```
@@ -0,0 +1,8 @@
+{
+  "type": "prerelease",
+  "comment": "SpinButton: reintroduce incorrect behavior of allowing updates when value is provided",
```

```suggestion
  "comment": "SpinButton: Reintroduce incorrect behavior of allowing updates when value is provided to prevent breaking partners.",
```

Just a suggestion because right now it would look weird on our release notes.